### DISCUSSION TITLE
More than 1 connection?

### DISCUSSION THREAD

**Discussion by dzungpv** [bertmelis/espMqttClient#127] [General]
I want feature like multiple connection to multiple mqtt server, seem not exist any lib support it. 
It is posible?
Now only official mqtt lib support multi connection https://docs.espressif.com/projects/esp-idf/en/latest/esp32/api-reference/protocols/mqtt.html

**Comment by bertmelis** [comment]
Did you test it? I didn't design with this requirement in mind but I also don't see any reason why it wouldn't work. Just create two clients and try!

**Reply by dzungpv** [reply]
OK I will try it

**Comment by Pablo2048** [comment]
It is working - I do have running two clients at the same time...
